src/refiners/fluxion/utils.py

```diff
@@ -8,30 +8,22 @@
 from PIL import Image
 from safetensors import safe_open as _safe_open  # type: ignore
 from safetensors.torch import save_file as _save_file  # type: ignore
-from torch import (
-    Tensor,
-    cat,
-    device as Device,
-    dtype as DType,
-    manual_seed as _manual_seed,  # type: ignore
-    no_grad as _no_grad,  # type: ignore
-    norm as _norm,  # type: ignore
-)
+from torch import Tensor, device as Device, dtype as DType
```

Comment:
Why are some device as Device (and dtype as DType) are replaced (torch.device, ...) and some others kept as-is? No way to be more consistent?

Reply:
:sweat_smile: I began to replace every `from torch import ...`, but saw that it would produce a very large diff (because of the `device as Device`, `dtype as DType`, ...), so I changed my mind and only modified the other imports like `cat`, `stack`, `zeros`, ... I completely forgot to revert the extra changes, let me fix this (I'll make a follow-up PR for these remnant imports to make everything more consistent globaly)
I'll also split the ic-light test fix in an other PR

Reply:
new PR for the IC-Light fix: #403